vue-frontend/src/pages/jobs/apply/[id].vue

```diff
@@ -531,6 +531,20 @@ export default {
       this.disableInput = false;
       this.isLoad = true;
 
+      //resume file name with the current date
+      var year = new Date().getFullYear();
+      var mon = new Date().toLocaleString("default", { month: "short" });
+      var date = new Date().getDate();
+
+      const currentDate = date + "-" + mon + "-" + year;
+
+      const file = this.file.name;
+
+      const splitFileName = file.split(".");
+
+      const fileName =
+        splitFileName[0] + "-" + currentDate + "." + splitFileName[1];
```

Comment:
Shouldn't this be a separate function?

Reply:
No, i don't think so create new function for that because updated file name is used over here only